pkgs/applications/misc/candle/default.nix

```diff
@@ -0,0 +1,34 @@
+{ stdenv, fetchFromGitHub, qtbase, qtserialport, qmake }:
+
+stdenv.mkDerivation rec {
+  name = "candle-${version}";
+  version = "1.1";
+
+  src = fetchFromGitHub {
+    owner  = "Denvi";
+    repo   = "Candle";
+    rev    = "v${version}";
+    sha256 = "1gpx08gdz8awbsj6lsczwgffp19z3q0r2fvm72a73qd9az29pmm0";
+  };
+
+  buildPhase = ''
+    cd src
+    qmake
+    buildPhase
+  '';
+
+  installPhase = ''
+    mkdir -p $out/bin
+    cp Candle $out/bin/
+  '';
+
+  buildInputs = [ qtbase qtserialport ];
+
+  meta = with stdenv.lib; {
+    description = "GRBL controller application with G-Code visualizer written in Qt";
+    homepage = https://github.com/Denvi/Candle;
+    license = licenses.gpl3;
+    maintainers = [ maintainers.matti-kariluoma ];
+    platforms = platforms.all;
```

Comment:
Superfluous

Reply:
I don't understand, which line should I delete? I infer platforms =

Reply:
Correct, ```platforms = platforms.all;``` can go.

Reply:
ok, removed squashed and pushed